ext/phar/phar.c

```diff
@@ -2796,10 +2794,7 @@ int phar_flush(phar_archive_data *phar, char *user_stub, zend_long len, int conv
 			return EOF;
 		}
 		newcrc32 = ~0;
-		mytime = entry->uncompressed_filesize;
-		for (loc = 0;loc < mytime; ++loc) {
-			CRC32(newcrc32, php_stream_getc(file));
-		}
+		crc32_stream_bulk_update(&newcrc32, file, entry->uncompressed_filesize);
 		entry->crc32 = ~newcrc32;
```

Comment:
Seems we don't need check the return here as it just generate a `crc` value here.

Reply:
This might be another plus to return the processed length and compare with `uncompressed_filesize` for a better security. Otherwise might make sense to `(void)` to silence possible static analyzer warnings.

thanks.